apps/lfx-one/src/app/modules/newsletters/components/newsletter-block-composer/newsletter-block-composer.component.scss

```
@@ -0,0 +1,175 @@
+
+.cdk-drag-animating {
+  transition: transform 200ms cubic-bezier(0, 0, 0.2, 1);
+}
+
+.cdk-drop-list-dragging .cdk-drag:not(.cdk-drag-placeholder) {
+  transition: transform 200ms cubic-bezier(0, 0, 0.2, 1);
+}
+
+// ---------------------------------------------------------------------------
+// Rendered-email preview classes.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)